triqler/hyperparameters.py

```diff
@@ -54,8 +67,16 @@ def fitPriors(peptQuantRows, params, printImputedVals = False, plot = False):
     imputedDiffs.extend((xImps - quantMatrixFiltered)[(~np.isnan(quantMatrixFiltered)) & (np.array(numNonNaNs) > 1)])
     #imputedVals.extend(xImps[(np.isnan(quantMatrixFiltered)) & (np.array(numNonNaNs) > 1)])
   
-  fitLogitNormal(observedXICValues, params, plot)
-  
+
+  # Add parameter to params for selecting fitLogitNormal
+  if params["missingValuePrior"] == "DIA":
+      peptide_group_means = np.array(peptide_group_means)
+      peptide_group_means = peptide_group_means[~np.isnan(peptide_group_means)]
+      fitLogitNormal(peptide_group_means, params, plot) # DIA fitLogitNormal - missing value prior
```

Comment:
I don't think you want to fit a logitNormal, you just want to fit a regular logit function.

Reply:
Aren'y we supposed to fit it the same function as the original, just with peptide means based on their missing values instead of all the observed XIC data?